tests/Unit/Services/GitPrMatcherServiceTest.php

```
@@ -151,18 +151,45 @@ public function testMatchAndLinkWithHighConfidenceMatch(): void
+                if (str_contains($sql, 'DELETE FROM ai_workflow_claims')) {
+                    $this->assertSame($claimLeaseToken, $params[':lease_token'] ?? null);
+                    $stmt->method('rowCount')->willReturn(1);
+                    return $stmt;
+                }
+
+                return $stmt;
+            }
+        );
 
         $gemini = $this->createMock(GeminiService::class);
         $gemini->method('generateJson')->willThrowException(new \RuntimeException('API error'));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '187', column '65'). (undefined)

(MissingImport)